@Ariana-B  the lint is failing

```
- hook id: yamllint
- exit code: 2

.github/workflows/publish-pypi.yml
  1:1       warning  missing document start "---"  (document-start)

```

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -40,7 +40,6 @@ repos:
     rev: v1.26.3
     hooks:
       - id: yamllint
-        args: [-s]
   # Common pre-commit checks
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.1.0

